src/Controller/FrontController.php

```diff
@@ -95,8 +95,9 @@ public function subscribed(ForumRepository $fr, SubmissionRepository $sr, string
      */
     public function all(ForumRepository $fr, SubmissionRepository $sr, string $sortBy, int $page) {
         # Added for v1.
+        $admin = !is_null($this->getUser()) && $this->getUser()->isAdmin();
```

Comment:
let's extract this method somewhere like
  ```
   static public function isAdmin($user) 
   { 
      return !is_null($this->getUser()) && $this->getUser()->isAdmin(); 
   }
```

so it can be used everywhere

Reply:
This is what happens when you work on stuff in VIM over a long period of time lol